packages/web-components/src/components/ic-hero/test/basic/__snapshots__/ic-hero.spec.ts.snap

```diff
@@ -1,14 +1,16 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`ic-hero component should render small: renders-small-variant 1`] = `
-<ic-hero class="ic-hero-small" heading="Test title" small="" subheading="Test text">
```

Comment:
I think this test needs updating in the spec.ts file to have size="small" now that the small prop has been removed

Reply:
Good spot! I've updated the test :)